inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/model/Predictions.java

```diff
@@ -261,4 +261,14 @@ public void removePredictions(Long recommenderId)
         predictions.entrySet()
             .removeIf((p) -> p.getKey().getRecommenderId() == recommenderId);
     }
+
+    public List<AnnotationObject> getPredictionsByTokenAndFeature(int aBegin, int aEnd,
```

Comment:
This method seems to have have a bug: it retrieves annotations at given position with a given feature name in **all** layers. I'm quite sure that is not what is intended. Make sure you disambiguate the feature with the layer.

Please add JavaDoc.